shake-bench/src/Development/Benchmark/Rules.hs

```diff
@@ -247,8 +250,8 @@ benchRules build benchResource MkBenchRules{..} = do
                 RemEnv "GHC_PACKAGE_PATH",
                 AddPath [takeDirectory ghcPath, "."] []
               ]
-              BenchProject{..}
-          cmd_ Shell $ "mv *.benchmark-gcStats " <> dropFileName outcsv
+              BenchProject {..}
+          cmd_ Shell $ "mv ghcide.hp " <> dropFileName outcsv </> dropExtension exp <.> "hp"
```

Comment:
Why use a shell command to do a move, rather than just renameFile?

Reply:
Fair enough, there's no longer a wildcard